.golangci.yml

```diff
@@ -303,4 +303,4 @@ issues:
   max-same-issues: 0
 
   # Show only new issues created after git revision `REV`
-  new-from-rev: origin/master
+  new-from-rev: 30e1c10d456af9d3778288cfb7c247ecd06a4339
```

Comment:
it might be nice to choose an older sha to verify the bug is actually fixed. could potentially go back to when we enabled the linter, although would need to fixup all the linting issues that snuck in.

Reply:
Good point, and I've actually tried that locally. The linter seems to be working with the old original from-rev (`5a3a12bba59e058b7a4cee6ec472c3f5786f741b`).
I'd prefer not to commit that though, because there seem to be a few new lint rules introduced, that have been violated a bunch of times during those few months, so I think it is better to have a new fresh start.